Scripts/s4ap/utils/s4ap_career_utils.py

```
@@ -0,0 +1,164 @@
+
+    @classmethod
+    def get_career_levels(cls, career_track: TunableCareerTrack, include_branches: bool = False) -> Tuple[CareerLevel]:
+        if career_track is None:
+            return tuple()
+        if include_branches:
+            if career_track is None:
+                return tuple()
+            if include_branches:
+                # noinspection PyUnresolvedReferences
+                if hasattr(career_track, 'career_levels') and career_track.career_levels is not None:
+                    career_levels: List[CareerLevel] = list(career_track.career_levels)
```

Ruff (0.14.7)

Undefined name `List`

(F821)